### DISCUSSION TITLE
Problem in providing the proof for Student Developer Pack

### DISCUSSION THREAD

**Discussion by gcazzola** [community/community#56401] [GitHub Education] [answered]
### Hi everyone,

Hi everyone, I am a student at Politecnico di Milano, Graduate School of Management, and I have been trying to apply for the GitHub Student Developer Pack. However, my application has been rejected several times and for the same reasons:

"Hey ******,
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 25, 2023 for @.
• You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
• Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

Once you have resolved the issues with your submission, visit us again and reapply.
Have questions? Join the GitHub Education Community for answers.
💖 The GitHub Education Team
"

I uploaded several documents but none of them looks like is eligible to get me approved. I have tried all the documents I could find but nothing works. Can someone help me? is there a support to whom i could send all the documentation?

I would really appreciate the help i could have from copilot especially for my master thesis. 

Please let me know

**Comment by gcazzola** [comment, marked as answer]
Apparently upload the files with the smartphone instead of pc made the difference guys

**Reply by tmgkopczyk** [reply]
How do I upload them though? It doesn't seem like there is a space for them on the app.

**Reply by sr5434** [reply]
It didn’t work for me

**Reply by theRealDadsdy** [reply]
What does that mean

**Reply by CodeBreaker518** [reply]
didn't work for me either

**Reply by Brioche23** [reply]
hi! What kind of documents did you upload? I cannot find any with a date in the Online services

**Reply by JnaneshaPS** [reply]
you hav eto upload yor school id card and school email id properly

**Comment by AwaisOem** [comment]
I am student at some university in Pakistan, our university don't provide us any email address.
I requested to officials and they refused to give email address to any student.
how can i apply without official email address?

Please let me know

**Reply by leon-wm** [reply]
Just apply with your normal E-Mail!
With your privat one.

**Reply by mhkasif** [reply]
![image](https://github.com/community/community/assets/30750191/a31503ba-1df5-48cf-ae38-affd7a6fbb77)
it keeps saying to add another email

**Reply by leon-wm** [reply]
Maybe if you add somethin like "**_IT_**" behind it or whatever you are studying, so the system will recognize it as a diffrent University, but you are still telling the truth.

Like for example:
`University of Karachi, IT faculty`

**Comment by TeoBale** [comment]
Hello I am a high school student in the institute iismachiavelli of Pioltello in the province of Milan (Italy).

I tried several times to follow the instructions to complete the appication for the student-developer-pack but unfortunately neither of the two times I was able to get it.

The first time I sent a photo of the report card that my school issues at the end of each year, the papers are official school papers and signed.

GitHub rightly replied that to verify my profile would require updating it with my real name and a new bio.

I followed their suggestion and entered my personal information on my profile with also a profile photo in which my face appeared

I tried to redo the application for the packet by sending a collage of photos containing me holding my school documents with my ID card and photos taken at the monitor on my electronic registry portal and on classroom where the school email required in the previous procedure also appeared.

Unfortunately, even doing so within 5 seconds I got an email saying that my information could not be verified even though on the ID card, which is an official document is underlined under "Profession": student

I hope someone can help me solve it, thank you in advance.... Matteo.

**Reply by leon-wm** [reply]
May try again with an ID/a document that wasn't given to you at end of your semester

**Comment by JnaneshaPS** [comment]
First you have to give an proper email given by the listed university and upload ID proof through the webcam which will be accepted easily

**Comment by mjsherwood** [comment]
I am currently trying to apply and my application was rejected as my location was not in the same geographic location as the school. I am studying in an online program so don't live near the school I am attending. I don't know how to get through this problem.

**Comment by gsckoco** [comment]
I can't apply for the student program since I am studying through the open university, and they do not provide any student ID, and I have had no correspondence from them via mail. Is there anything else I can use to prove I am a student or am I out of luck?

**Reply by ToadOnbekend** [reply]
You can always request the proof you're on the university.

**Comment by UsamaAbbasi1999** [comment]
Hi All,

I'm a working professional and also enrolled in Business Administration and I need a GitHub student developer pack to access the datacamp to learn the Data Analyst course. 

I'm having a problem that I've applied for admission and it may take up to almost 1 - 2 months for me to get properly enrolled in the program. Until then lots of time will be wasted.

Is there anyone who can help me in this regard? I don't know what kind of help people can do but I really need access to data camp to take career course of Data Analyst.

**Comment by rahulc0dy** [comment]
My university does not issue any email addresses to students but the students developer program application asks to provide a university issued email. Now  I cannot Renew my application for github students developer pack

**Reply by NovigradWitcher** [reply]
Hi, did you find any solution? Stuck in the same boat as you, even opened two tickets 2 weeks back but GitHub hasn't even replied

**Comment by nandasafiqalfiansyah** [comment]
![Screenshot_2023-08-19-16-26-38-894_com.android.chrome.jpg](https://github.com/community/community/assets/101439204/adf1c313-21e3-406c-b24d-92f80b87a40a)

Hi im student for indonesia plz help me 😭

**Reply by masshitro** [reply]
Hi,allcan you  help me please. I am a student of a university located in Ukraine, and our corporate email has not been working for a year. However, only the corporate email is accepted for verification. How can I pass verification by providing my student ID and Gmail, for example?
